command/connect/envoy/envoy.go

```diff
@@ -551,7 +551,7 @@ func (c *cmd) grpcAddress(httpCfg *api.Config) (GRPC, error) {
 	} else {
 		// Parse as host:port with option http prefix
 		grpcAddr = strings.TrimPrefix(addr, "http://")
-		grpcAddr = strings.TrimPrefix(addr, "https://")
```

Comment:
This looks like a real bug. I missed it while working on #7608. `http://` is never stripped because the second line overrides any work done by the first.

Reply:
Good catch.

Reply:
Should the fix here instead be `grpcAddr = strings.TrimPrefix(grpcAddr, "https://")` to operate on the result of the first line (stripping `http://` or `https://`) instead of removing?

Reply:
Exactly! That's the change I made. I guess by commenting on the removed line it shows up weird in the comment summary.